distribution_metadata.c

```diff
@@ -594,16 +543,11 @@ TupleToShardPlacement(HeapTuple heapTuple, TupleDesc tupleDescriptor)
 	ShardPlacement *shardPlacement = NULL;
 	bool isNull = false;
 
-	Datum idDatum = heap_getattr(heapTuple, ATTR_NUM_SHARD_PLACEMENT_ID,
```

Comment:
They're no longer tied to the table definition, but rather to the query order. As here is another place where the query lives in a separate function from the code extracting values, I see that preserving them could make sense. I'll put them back, I think…